corehq/apps/es/utils.py

```diff
@@ -32,3 +35,61 @@ def flatten_field_dict(results, fields_property='fields'):
             new_val = val[0]
         field_dict[key] = new_val
     return field_dict
+
+
+def chunk_query(query, term, chunk_size=1000):
+    query = deepcopy(query)
+
+    path_to_term = _chunk_query([], None, query.es_query, term)
+    chunked_queries = []
+
+    # No path to term could be found, return just the query
+    if path_to_term is None:
+        return [query]
+
+    term_list = _dict_lookup(path_to_term, query.es_query)
+
+    # Cannot chunk term that is not a list
+    if not isinstance(term_list, list):
+        return [query]
```

Comment:
good question, i think it could be chunked if there wasn't an OR filter (would only be one filter), but we certainly don't want to do it to an AND filter. don't think it'd be too hard to check that